app/Generators/views/super-hero.twig

```diff
@@ -65,12 +65,11 @@
           {{ post_date }}
         </time>
 
-        <section class="op-ad-template">
-          <!-- Ads to be automatically placed throughout the article -->
-          {% for advert in adverts.body %}
-            {{ advert }}
-          {% endfor %}
-        </section>
+        <figure class="op-ad">
+            {% for advert in adverts %}
```

Comment:
Ah, you can't have multiple iframes inside a `<figure class="op-ad">`. You could when using `<section class="op-ad-template">` and you were creating multiple `<figure class="op-ad">`